logging/cluster-logging-external.adoc

```diff
@@ -41,7 +41,50 @@ include::modules/cluster-logging-collector-log-forward-syslog.adoc[leveloffset=+
 
 include::modules/cluster-logging-collector-log-forward-cloudwatch.adoc[leveloffset=+1]
 
+=== Forwarding logs to Amazon CloudWatch from STS enabled clusters
```

Comment:
Should this be a procedure module?
```suggestion
=== Forwarding logs to Amazon CloudWatch from STS-enabled clusters
```

Reply:
If we were allowed to use xrefs in modules it absolutely would be. Since we aren't, its in the assembly.